lib/gcp/cloudfunctions.js

```diff
@@ -27,6 +27,7 @@ function _createFunction(options) {
   var endpoint = '/' + API_VERSION + '/' + location + '/functions';
   var data = {
     sourceArchiveUrl: options.sourceArchiveUrl,
+    sourceUploadUrl: options.sourceUploadUrl,
```

Comment:
Why do we need both sourceArchiveUrl and sorceUploadUrl?

Reply:
forgot to remove this line!